dependencyManagement/build.gradle.kts

```diff
@@ -11,20 +11,14 @@ data class DependencySet(val group: String, val version: String, val modules: Li
 val dependencyVersions = hashMapOf<String, String>()
 rootProject.extra["versions"] = dependencyVersions
 
-// this line is managed by .github/scripts/update-sdk-version.sh
-val otelSdkVersion = "1.19.0"
-// this line is managed by .github/scripts/update-instrumentation-version.sh
-val otelInstrumentationVersion = "1.19.0"
```

Comment:
there seem to be some situations where dependabot understands version variables and some that it doesn't, but I gave up 🤷‍♂️